packages/frontend/app/pods/components/pod/application/off-canvas-container/sidebar/contacts/component.ts

```diff
@@ -25,12 +25,13 @@ export default class ContactsSidebar extends Component<IArgs> {
   }
 
   get contacts() {
+    const sortedContacts = this.allContacts.sort((a, b) =>
+      a.isPinned === b.isPinned ? 0 : a.isPinned ? -1 : 1);
     if (!this.hideOfflineContacts) {
-      return this.allContacts;
+      return sortedContacts;
     }
-
-    return this.allContacts.filter(contact => {
-      return contact.onlineStatus !== STATUS.OFFLINE;
+    return sortedContacts.filter(contact => {
+      return contact.onlineStatus !== STATUS.OFFLINE || contact.isPinned;
```

Comment:
I wonder if we should extract an 'shouldBeInSidebar' helper function, an well as the opposite (for below)